modules/profile/manifests/tomcat.pp

```diff
@@ -1,26 +1,32 @@
-class profile::tomcat (  
-    $dports         = ['80', '8080', '8080'],
-    $ssh_user       = 'if083',
-    $ssh_group      = 'wheel',
-    $ssh_password   = 'derferterela',
-) {
-
+class profile::tomcat(
+  String $man_user       = 'manager',  
```

Comment:
Why we have variable type specified here?
But in other place no String List or Integers?
It should be consistent across all classes.